packages/datastore/datastore/src/storage/adapters/WebSQLAdapter.ts

```diff
@@ -231,12 +231,25 @@ const prepareStatement = (input: any, type: string = "insert"): [string, any[]]
   invariant(false, "Unsupported query type");
 };
 
-const predicateToSQL = (predicate: ModelFieldPredicate) => {
-  const key = predicate.getKey();
-  const op = predicate.getOperator().op;
-  const operator = (op === "eq") ? "=" : undefined;
-  invariant(operator, "Operator not supported");
-  return `WHERE ${key} ${operator} ?`;
+const filterToSQL = (filter?: Filter) => {
+  if (!filter) { return ""; };
+
+  const tokens: string[] = [];
+
+  Object.keys(filter).forEach((key) => {
+    if (!(filter[key] instanceof Object)) {
+      tokens.push(`eq=${filter[key]}`);
+      return;
+    }
+    const op = Object.keys(filter[key])[0];
+    const operator = (op === "eq") ? "=" : undefined;
+    invariant(operator, "Operator not supported");
+
+    const value = filter[key][op];
+    tokens.push(`${key}=${value}`);
+  });
+
+  return `WHERE ${tokens.join(" AND ")} ?`;
```

Comment:
I guess we do not care about SQL Injection here?

Reply:
I only dropped base work for SQL support